msteams-platform/tabs/how-to/authentication/auth-aad-sso.md

```diff
@@ -101,18 +100,17 @@ This section describes the tasks involved in creating a Teams tab that uses SSO.
 
 1. Navigate to **Authentication**.
 
-    If an app has not been granted IT admin consent, users have to provide consent the first time they use an app.
+    If an app hasn't been granted IT admin consent, users have to provide consent the first time they use an app.
```

Comment:
This part is very important, but it's buried in a section about configuring the AAD application.